tests/pkg_cryptoauthlib_internal-tests/Makefile

```diff
@@ -0,0 +1,10 @@
+include ../Makefile.tests_common
+
+INCLUDES += -I$(RIOTBASE)/pkg/cryptoauthlib/include
+FEATURES_REQUIRED += periph_i2c
+CFLAGS += -O3 -Wno-unused-parameter -Wno-missing-field-initializers -Wno-int-conversion -Wno-unused-function -Wno-unused-variable
+CFLAGS += -DATCA_HAL_I2C -DDO_NOT_TEST_CERT
```

Comment:
Same as above regarding `CFLAGS` and `INCLUDES`: Fix what can be fixed so we won't need to set all these compiler flags. For what can't be fixed right away we should set respective flags/includes under pkg/cryptoauthlib.

Reply:
At least the `ATCA_HAL_I2C` should be redundant now, tight?